### PR TITLE
finish result-componen

### PR DESCRIPTION
**Pull request by mohamadhase**

### CODE REVIEW

File: front-end/Image-recognition/src/app/result-component/result-component.component.ts [hunk 1]
```diff
@@ -1,61 +1,156 @@
...
+  showTags = true;
+  showText = true;
+  isWordCloudExpanded = false;
+  canvas: HTMLCanvasElement;
+  context: CanvasRenderingContext2D;
+  private labels: string[] = [ ]; //  label names array
+
+  private scores: number[]= []; // scores array
+  private text_response :TextResponse[]= []
+@ViewChild('myCanvas', { static: true }) canvasRef: ElementRef<HTMLCanvasElement>;
```
Comment (Reema-Khaseeb) [inline front-end/Image-recognition/src/app/result-component/result-component.component.ts]: Unused

File: front-end/Image-recognition/src/app/result-component/result-component.component.ts [hunk 2]
```diff
@@ -1,61 +1,156 @@
...
-  ]; // scores array
+
+export class ResultComponentComponent implements  AfterViewInit{
+  constructor(private route: ActivatedRoute) { }
+
+  showTags = true;
+  showText = true;
+  isWordCloudExpanded = false;
+  canvas: HTMLCanvasElement;
+  context: CanvasRenderingContext2D;
```
Comment (Reema-Khaseeb) [inline front-end/Image-recognition/src/app/result-component/result-component.component.ts]: Unused attributes

**Review by Reema-Khaseeb — CHANGES_REQUESTED** [review]

File: front-end/Image-recognition/src/app/result-component/result-component.component.ts (hunk 1 above)
Comment (mohamadhase) [reply front-end/Image-recognition/src/app/result-component/result-component.component.ts]: Done

File: front-end/Image-recognition/src/app/result-component/result-component.component.ts (hunk 2 above)
Comment (mohamadhase) [reply front-end/Image-recognition/src/app/result-component/result-component.component.ts]: deleted them

**Pull request merged**